src/SYS.ASM

```diff
@@ -338,50 +323,23 @@ command:                                        ; Main loop: Retrieve command fr
     jmp command                                 ; Get a new command
 
 .input:
-    push di                                     ; Preserve destination index
-
     cmp byte [si], 0x3C                         ; Is the byte another angle bracket?
     je .immediate                               ; If so, it may be immediate input
 
     cmp byte [si], 0x20                         ; Is the byte a space?
     jne .bad_command                            ; If not, bad command
 
     inc si                                      ; Otherwise, ignore space
+    mov di, misc_buffer                         ; Miscellaneous buffer
+    call get_raw_param                          ; Get parameter from input buffer
```

Comment:
The same parameter-grabbing routine is used for getting filenames/words from special modifiers.